core/kt-osrd-utils/src/main/kotlin/fr/sncf/osrd/utils/units/Speed.kt

```diff
@@ -0,0 +1,5 @@
+package fr.sncf.osrd.utils.units
+
+
+@JvmInline
+value class Speed(val value: ULong)
```

Comment:
This seems unfinished. `mps`, `kmph` conversions? `PrimitiveWrapperCollection` generation?

Also, the property should probably be called `ms`/`mps`/`metersPerSecond`, not `value`

Reply:
It has been moved from `RawSignalingInfra`, there's no real change there.

I assume @multun moved it there to later use it in speed limits (which haven't been added yet). But using doubles would be easier for the migration, we can move to a dedicated speed class later on.

Reply:
I would very much like to try to stick to wrapped integers, to avoid the flock of issues we had with float arithmetics

Reply:
Let's keep it then, but I'll add some stuff to the class to make the unit explicit. I don't handle speed yet in this PR so it doesn't actually change anything here, the speed data are still in `//TODO` form. 

We need to figure out a good precision tough, depending on the time step we may have troubles with slow speedup. I believe using millimeters and milliseconds will only work as long as we use a "macro" time step (a few seconds). Which we currently do (2s), but I don't know if that fit every use case for the application. 

Apparently the minimum value for train acceleration is [3cm/s^-2](https://www.sncf-reseau.com/sites/default/files/2019-06/Donn%C3%A9es%20n%C3%A9cessaires%20au%20calcul%20des%20marches%2C%20%C3%A0%20la%20d%C3%A9termination%20des%20charges%20remorquables%20et%20des%20capacit%C3%A9s%20d%27acc%C3%A9l%C3%A9ration.pdf).

Reply:
LGTM